From 436fb8443d69f21476f1bda6789f107006344cd7 Mon Sep 17 00:00:00 2001
From: Xu Han <xu.han@intel.com>
Date: Mon, 25 Jan 2016 13:17:17 +0800
Subject: [PATCH 320/403] Change PPAT register handler method from F_PT to
 F_DOM0

* This patch is fix missed PPAT register handler method change.
* It was reviewed at bc branch, and missed change it when
  porting to staging branch.

Signed-off-by: Xu Han <xu.han@intel.com>
---
 drivers/gpu/drm/i915/vgt/handlers.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/vgt/handlers.c b/drivers/gpu/drm/i915/vgt/handlers.c
index 3254a1a..0dc5981 100644
--- a/drivers/gpu/drm/i915/vgt/handlers.c
+++ b/drivers/gpu/drm/i915/vgt/handlers.c
@@ -3707,8 +3707,8 @@ reg_attr_t vgt_reg_info_bdw[] = {
 
 {0x1C054, 4, F_DOM0, 0, D_BDW_PLUS, NULL, NULL},
 /* BDW */
-{GEN8_PRIVATE_PAT_LO, 4, F_PT, 0, D_BDW_PLUS, NULL, gen8_ppat_write},
-{GEN8_PRIVATE_PAT_HI, 4, F_PT, 0, D_BDW_PLUS, NULL, gen8_ppat_write},
+{GEN8_PRIVATE_PAT_LO, 4, F_DOM0, 0, D_BDW_PLUS, NULL, gen8_ppat_write},
+{GEN8_PRIVATE_PAT_HI, 4, F_DOM0, 0, D_BDW_PLUS, NULL, gen8_ppat_write},
 
 {GAMTARBMODE, 4, F_DOM0, 0, D_BDW_PLUS, NULL, NULL},
 
-- 
1.7.10.4

